examples-nosql-spring-sdk/tutorial-combining-java-sdk-and-spring-sdk-composite-key/README.md

```diff
@@ -0,0 +1,45 @@
+# Preparation
+
+Clone this repository
+
+# Setup dependencies
+
```

Comment:
Java JDK tested version and maven version would be useful if users run into errors. Maybe say something about which OSes it's expected to run.